src/metabase/search/postgres/index.clj

```diff
@@ -175,10 +175,12 @@
   [search-term]
   {:select [:model_id :model]
    :from   [active-table]
-   :where  [:raw
-            "search_vector @@ to_tsquery('"
-            tsv-language "', "
-            [:lift (to-tsquery-expr search-term)] ")"]})
+   :where  (if-not search-term
+             [:= [:inline 1] [:inline 1]]
```

Comment:
I'm not sure we want to keep supporting this when we stabilize this search engine, but for now its good to keep parity with the existing engine's behavior when there is no query.